src/purgatory/ToggleFiltersButton.js

```diff
@@ -2,7 +2,7 @@ import React from 'react'
 import { withTheme } from '../utils/theme'
 
 let ToggleFiltersButton = ({ onClick, theme: { AlternateButton, Icon } }) => (
-  <AlternateButton title="Toggle Filters" onClick={onClick}>
+  <AlternateButton title="Show Filters" onClick={onClick}>
```

Comment:
A generic `ToggleFiltersButton` shouldn't assume that it's always `Show`. Even that happens to be true in our use cases, that's not the assumption you'd make for this. Ideally, we'd make it dynamic even though this currently doesn't know about the current value of show/hide.

Reply:
After talking through this, the implementation would get really complicated because mode is actually a 3 way toggle. Let just call this `ShowFiltersButton` instead so it's super clear. This is, in theory, a breaking change - but real end users would just use `ToggleFiltersHeader` so this component never really needed to be exported anyway. We'll pretend like it never was 😄